README.md

```diff
@@ -24,11 +24,11 @@ julia> map(modifying(a = string), data)
 julia> reduce(modifying(a = +), data)
 (a = 6, b = 7)
 
-julia> using Setfield: @lens
+julia> using Accessors: @optic
```

Comment:
This is confusing. It seems that somehow CI is using the old `README.md`.

Reply:
I think there was a problem in updating the `Manifest.toml` files. I guess my setup is a bit tricky because I have `Manifest.toml`s checked in.